### PR TITLE
Fixed RemoteTestResultTest tests without tblib.

### PR DESCRIPTION
**Pull request by felixxm**
Follow up to e3bca22e7e572b0274a0814c7869c899d7b544e0.

See [logs](https://djangoci.com/job/main-no-requirements/106/).

@adamchainz Can you take a look?

### CODE REVIEW

File: tests/test_runner/test_parallel.py [hunk 1]
```diff
@@ -63,22 +70,24 @@ def test_was_successful_one_success(self):
...
         result = RemoteTestResult()
-        result.addError(None, ValueError('woops'))
+        result.addError(None, self._test_error_exc_info())
         self.assertIs(result.wasSuccessful(), False)
 
+    @unittest.skipUnless(tblib is not None, 'requires tblib to be installed')
     def test_was_successful_one_failure(self):
         result = RemoteTestResult()
-        result.addFailure(None, ValueError('woops'))
+        result.addFailure(None, self._test_error_exc_info())
```
Comment (felixxm) [inline tests/test_runner/test_parallel.py:90]: `addFailure()`, `addError()`, and `addExpectedFailure()` expect `exc_info`.

File: tests/test_runner/test_parallel.py [hunk 2]
```diff
@@ -63,22 +70,24 @@ def test_was_successful_one_success(self):
...
         self.assertIs(result.wasSuccessful(), True)
 
+    @unittest.skipUnless(tblib is not None, 'requires tblib to be installed')
     def test_was_successful_one_error(self):
         result = RemoteTestResult()
-        result.addError(None, ValueError('woops'))
+        result.addError(None, self._test_error_exc_info())
         self.assertIs(result.wasSuccessful(), False)
 
+    @unittest.skipUnless(tblib is not None, 'requires tblib to be installed')
```
Comment (felixxm) [inline tests/test_runner/test_parallel.py:87]: This test crashes (as expected) without `tblib`:
```
  File "/django/django/test/runner.py", line 164, in _confirm_picklable
    pickle.loads(pickle.dumps(obj))
TypeError: cannot pickle 'traceback' object
```

File: tests/test_runner/test_parallel.py [hunk 3]
```diff
@@ -63,22 +70,24 @@ def test_was_successful_one_success(self):
...
-        result.addExpectedFailure(None, ValueError('woops'))
+        result.addExpectedFailure(None, self._test_error_exc_info())
         self.assertIs(result.wasSuccessful(), True)
 
     def test_was_successful_one_skip(self):
         result = RemoteTestResult()
         result.addSkip(None, 'Skipped')
         self.assertIs(result.wasSuccessful(), True)
 
+    @unittest.skipUnless(tblib is not None, 'requires tblib to be installed')
```
Comment (felixxm) [inline tests/test_runner/test_parallel.py:81]: This test crashes (as expected) without `tblib`:
```
  File "/django/django/test/runner.py", line 164, in _confirm_picklable
    pickle.loads(pickle.dumps(obj))
TypeError: cannot pickle 'traceback' object
```

**Review by carltongibson — APPROVED** [review]
Yep, perfect. Thanks @felixxm!

**Comment by felixxm** [comment]
@carltongibson Thanks for checking :+1:

**Comment by adamchainz** [comment]
Sorry I didn't respond, but yes LGTM

**Pull request merged**